ecommerce/frontend/src/components/VendorContent/VendorSplash.js

```diff
@@ -0,0 +1,92 @@
+import { useAppContext } from '../../context/AppContext'
+import React, { useEffect, useState } from 'react'
+import { Spin, Button, Rate, Switch } from 'antd'
+import { api } from '../../api'
+import { useHistory } from 'react-router-dom'
+import { EditOutlined } from '@ant-design/icons'
+import { round } from '../../utils'
+import './VendorSplash.less'
+
+const getVendorRatingLevel = ({ rating }) => {
+    if (rating <= 5.0) {
+        return 'low'
+    }
+    if (rating <= 8.0) {
+        return 'medium'
+    }
+    return 'high'
+}
```

Comment:
In the updated master branch I exported this function from utils. Please use it from there

Reply:
done